auto_nag/scripts/crash_samll_volume.py

```diff
@@ -0,0 +1,188 @@
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+
+        existing_keywords = {
+            keyword
+            for keyword in ("topcrash", "topcrash-startup")
+            if keyword in bug["keywords"]
+        }
+
+        signatures = utils.get_signatures(bug["cf_crash_signature"])
+
+        top_crash_signatures = [
+            signature
+            for signature in signatures
+            if signature in self.topcrash_signatures
+        ]
+
+        keep_topcrash_startup = any(
+            any(
+                criterion["is_startup"]
+                for criterion in self.topcrash_signatures[signature]
+            )
+            for signature in top_crash_signatures
+        )
+
+        keywords_to_remove = None
+        if not top_crash_signatures:
+            keywords_to_remove = existing_keywords - {"topcrash", "topcrash-startup"}
+        elif not keep_topcrash_startup:
+            keywords_to_remove = existing_keywords - {"topcrash-startup"}
+
+        data[bugid] = {
+            "severity": bug["severity"],
+            "keep_topcrash": bool(top_crash_signatures),
+            "keywords_to_remove": keywords_to_remove,
+            "signatures": signatures,
+        }
+
+        return bug
+
+    def _get_low_volume_crash_signatures(self, bugs):
+        signatures = {
+            signature
+            for bug in bugs.values()
+            if not bug["keep_topcrash"] and bug["severity"] in HIGH_SEVERITY
+            for signature in bug["signatures"]
+        }
+
+        signature_volume = Topcrash().fetch_signature_volume(signatures)
+
+        low_volume_signatures = {
+            signature for signature, volume in signature_volume.items() if volume < 5
+        }
+
+        return low_volume_signatures
+
+    def get_bugs(self, date="today", bug_ids=..., chunk_size=None):
+        bugs = super().get_bugs(date, bug_ids, chunk_size)
+        self.set_autofix(bugs)
+
+        # Keep only bugs with an autofix
+        bugs = {
+            bugid: bug for bugid, bug in bugs.items() if bugid in self.autofix_changes
+        }
+
+        return bugs
+
+    def set_autofix(self, bugs):
+        """Set the autofix for each bug."""
+
+        low_volume_signatures = self._get_low_volume_crash_signatures(bugs)
+
+        for bugid, bug in bugs.items():
+            autofix = {}
+            reasons = []
+            if bug["keywords_to_remove"]:
+                reasons.append(
+                    "The crash signature linked to this bug is not a topcrash signature anymore."
+                    if len(bug["signatures"]) == 1
+                    else "The crash signatures linked to this bug are not in the topcrash signatures anymore."
+                )
+                autofix["keywords"] = {"remove": list(bug["keywords_to_remove"])}
+
+            if bug["severity"] in HIGH_SEVERITY and all(
+                signature in low_volume_signatures
+                or signature in self.blocked_signatures
```

Comment:
If a crash signature is among the general signatures that we ignore when setting the `topcrash` keywords (e.g. `OOM | small`), we treat it the same as a low-volume crash. 

<details>
<summary>Click here to see the full list of blocked signatures!</summary><br>

- `java.lang.OutOfMemoryError: at java.lang.reflect.Array.createObjectArray(Native Method)`
- `OOM | large | mozalloc_abort | webrender::hit_test::HitTestingScene::new`
- `OOM | large | mozalloc_abort | rkv::backend::impl_safe::environment::{{impl}}::open`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_reuse_items`
- `OOM | large | mozglue.dll`
- `OOM | large | xul.dll | BaseThreadInitThunk`
- `java.lang.OutOfMemoryError: at java.io.ByteArrayOutputStream.<init>(ByteArrayOutputStream.java)`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.GeckoAppShell.getProxyForURI(GeckoAppShell.java)`
- `OOM | large | mozalloc_abort | xul.dll | _PR_NativeRunThread | pr_root`
- `OOM | unknown | NS_ABORT_OOM | nsGenericHTMLElement::GetURIAttr`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementsInternal<T> | CCGraphBuilder::NoteWeakMapping`
- `OOM | large | libxul.so@0x5aca8a | libxul.so@0xa305fff | libxul.so@0xa305fff | libxul.so@0x109034b | i965_dri.so@0x279a18`
- `java.lang.OutOfMemoryError: at mozilla.components.support.ktx.kotlinx.coroutines.flow.FlowKt$filterChanged$1.invokeSuspend(Flow.kt:12)`
- `OOM | large | mozalloc_abort | xul.dll`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::scache::NewBufferFromStorageStream`
- `java.lang.OutOfMemoryError: at android.util.Log.getStackTraceString(Log.java)`
- `OOM | large | mozalloc_abort | webrender::lru_cache::LRUCache<T>::push_new<T>`
- `OOM | unknown | NS_ABORT_OOM | mozilla::dom::URL::GetHref`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::Animation::QueuePlaybackEvent`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsMsgBodyHandler::ApplyTransformations`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::Zone::NewArray<T>`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::ArgumentsObject::objectMoved`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash`
- `OOM | large | xul.dll | InternalCallWinProc`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF8toUTF16::NS_ConvertUTF8toUTF16 | nsPrimitiveHelpers::CreatePrimitiveForCFHTML`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::dom::CallbackObject::~CallbackObject`
- `OOM | unknown | mozilla::dom::JSActorManager::ReceiveRawMessage`
- `java.lang.OutOfMemoryError: at java.lang.Thread.nativeCreate(Native Method)`
- `OOM | large | NS_ABORT_OOM | mozilla::extensions::URLInfo::Spec`
- `java.lang.OutOfMemoryError: at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.setIndexInArray(CoroutineScheduler.kt:1)`
- `OOM | large | mozalloc_abort | moz_xrealloc | xul.dll | _PR_NativeRunThread | pr_root`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::dom::FragmentOrElement::cycleCollection::Unlink`
- `OOM | large | NS_ABORT_OOM | nsTextFormatter::StringStuff`
- `java.lang.OutOfMemoryError: at java.lang.Daemons$FinalizerWatchdogDaemon.finalizerTimedOut(Daemons.java)`
- `IPCError-browser | ShutDownKill`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::define_clip_rounded_rect`
- `OOM | large | NS_ABORT_OOM | mozilla::layers::SharedSurfacesParent::Add`
- `OOM | large | mozalloc_abort | webrender::prim_store::gradient::GradientGpuBlockBuilder::build`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.mozglue.GeckoLoader.nativeRun(Native Method)`
- `OOM | large | EMPTY: no crashing thread identified; EmptyMinidump`
- `java.lang.OutOfMemoryError: at kotlin.ResultKt.createFailure(Result.kt:1)`
- `java.lang.OutOfMemoryError: at android.util.ArrayMap.allocArrays(ArrayMap.java)`
- `java.lang.OutOfMemoryError: at androidx.constraintlayout.solver.SolverVariable.<init>(SolverVariable.java)`
- `java.lang.OutOfMemoryError: at java.lang.String.substring(String.java)`
- `OOM | large | firefox-esr@0x17ef8`
- `java.lang.OutOfMemoryError: at java.lang.reflect.Constructor.newInstance0(Native Method)`
- `OOM | large | mozalloc_abort | xul.dll | cert_storage::make_env | cert_storage::SecurityState::open_db | cert_storage::{{impl}}::allocate::GetRevocationState | mozilla::psm::NSSCertDBTrustDomain::GetCertTrust`
- `OOM | large | __delayLoadHelper2 | _tailMerge_dbghelp.dll | MozWalkTheStack`
- `OOM | large | xul.dll | <unknown in ucrtbase.dll> | xul.dll | AesRNGState_generate`
- `OOM | large | mozalloc_abort | moz_xrealloc | SkTDArray<T>::adjustCount`
- `OOM | large | NS_ABORT_OOM | nsJSUtils::GetCallingLocation`
- `java.lang.OutOfMemoryError: at java.lang.Float.valueOf(Float.java)`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::SetLength<T> | mozilla::MediaTrackGraphImpl::OneIterationImpl`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::XMLHttpRequestMainThread::Open`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodePeepholeOptimization::OptimizeBytecode`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::HeapPtr<T>::postBarrieredSet`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_stacking_context`
- `OOM | large | NS_ABORT_OOM | mozilla::storage::Statement::GetString`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::finish_item_group`
- `OOM | large | NS_ABORT_OOM | MimeInlineTextHTMLParsed_parse_line`
- `EMPTY: no crashing thread identified; MissingSystemInfo`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::HTMLInputElement::SetValue`
- `OOM | large | NS_ABORT_OOM | mozilla::safebrowsing::LookupCache::GetLookupFragments`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::finalize`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Append | nsTextFormatter::StringStuff`
- `OOM | large | mozalloc_abort | moz_xmalloc | IPC::Message::IPDLMessage`
- `java.lang.OutOfMemoryError: at android.os.Parcel.nativeReadString(Native Method)`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | std::basic_streambuf<T>::xsputn`
- `OOM | unknown | NS_ABORT_OOM | NS_ConvertUTF16toUTF8::NS_ConvertUTF16toUTF8 | nsContentUtils::GetAttrTriggeringPrincipal`
- `java.lang.OutOfMemoryError: at com.android.org.conscrypt.ConscryptFileDescriptorSocket.transitionTo(ConscryptFileDescriptorSocket.java)`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::FragmentOrElement::CanSkip`
- `java.lang.OutOfMemoryError: at java.util.LinkedHashMap.addNewEntry(LinkedHashMap.java)`
- `OOM | large | mozalloc_abort | xul.dll | std::collections::hash::map::HashMap<T>::insert<T> | fog::private::timing_distribution::impl$3::start`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Append | mozilla::dom::FileReaderSync::ReadAsText`
- `java.lang.OutOfMemoryError: at kotlinx.coroutines.channels.AbstractChannel.receiveSuspend(AbstractChannel.kt:3)`
- `OOM | large | mozalloc_abort | static nsINode::operator new`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::with_capacity`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::StoreBuffer::GenericBuffer::put<T>`
- `java.lang.OutOfMemoryError: at java.util.LinkedHashMap.newValueIterator(LinkedHashMap.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::MergeRealms`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF8toUTF16::NS_ConvertUTF8toUTF16 | nsContentSecurityManager::MeasureUnexpectedPrivilegedLoads`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | InnerTextAccumulator::Append`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::MakeEntryHandle | mozilla::BaseHistory::ScheduleVisitedQuery`
- `OOM | large | mozalloc_abort | moz_xmalloc | XPCConvert::JSData2Native`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::gfx::SourceSurfaceRawData::GuaranteePersistance`
- `java.lang.OutOfMemoryError: at androidx.collection.SimpleArrayMap.allocArrays(SimpleArrayMap.java)`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::HTMLMediaElement::GetCurrentSrc`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::MapObject::setWithHashableKey`
- `OOM | large | mozalloc_abort | IPC::ParamTraits<mozilla::ipc::ByteBuf>::Read`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | PostWriteBarrier`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::collectToObjectFixedPoint`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::ipc::IPDLParamTraits<T>::Read`
- `OOM | large | mozalloc_abort | rkv::backend::impl_safe::environment::impl$0::open`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsFlexContainerFrame::DoFlexLayout`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::MakeEntryHandle | PLDHashTable::WithEntryHandle<T> | mozilla::dom::FragmentOrElement::cycleCollection::CanSkipInCCReal`
- `java.lang.OutOfMemoryError: at java.util.concurrent.ConcurrentLinkedQueue.newNode(ConcurrentLinkedQueue.java)`
- `OOM | large | mozalloc_abort | xul.dll | NS_internal_main`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::dom::CharacterData::cycleCollection::Unlink`
- `java.lang.OutOfMemoryError: at com.sun.jna.Structure.<init>(Structure.java)`
- `OOM | unknown | NS_ABORT_OOM | mozilla::jni::StringParam::StringParam`
- `OOM | large | mozalloc_abort | moz_xrealloc | xul.dll`
- `java.lang.OutOfMemoryError: at java.lang.AbstractStringBuilder.enlargeBuffer(AbstractStringBuilder.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::NativeIterator::NativeIterator`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::Zone::New<T>`
- `OOM | large | NS_ABORT_OOM | NS_MakeAbsoluteURI`
- `OOM | large | NS_ABORT_OOM | nsTPromiseFlatString<T>::Init`
- `OOM | large | EMPTY: no crashing thread identified; MissingThreadList`
- `OOM | large | mozalloc_abort | __rg_oom`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::StoreBuffer::WholeCellBuffer::allocateCellSet`
- `OOM | unknown | NS_ABORT_OOM | nsIFrame::DisplayBorderBackgroundOutline`
- `OOM | large | mozalloc_abort | rkv::backend::impl_safe::transaction::{{impl}}::commit`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::CharacterData::GetData`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | nsDataHandler::CreateNewURI`
- `OOM | large | mozalloc_abort | std::fs::read::inner`
- `java.lang.OutOfMemoryError: at org.mozilla.thirdparty.com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java)`
- `OOM | large | NS_ABORT_OOM | mozilla::ContentEventHandler::GenerateFlatTextContent`
- `OOM | large | mozalloc_abort | moz_xmalloc | ots::Arena::Allocate`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::vector<T>::_Resize<T>`
- `OOM | large | mozalloc_abort | moz_xrealloc | xul.dll | wmain`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | mozilla::HashMap<T>::put<T>`
- `OOM | large | mozalloc_abort | webrender::prepare::prepare_interned_prim_for_render`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::NativeDefineProperty`
- `OOM | unknown | NS_ABORT_OOM | NS_ConvertUTF8toUTF16::NS_ConvertUTF8toUTF16 | NS_GetSanitizedURIStringFromURI`
- `java.lang.OutOfMemoryError: at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.addConditionWaiter(AbstractQueuedSynchronizer.java)`
- `java.lang.OutOfMemoryError: at java.nio.HeapByteBuffer.<init>(HeapByteBuffer.java)`
- `OOM | unknown | NS_ABORT_OOM | nsTHashtable<T>::PutEntry | nsBlockFrame::AddFrames`
- `OOM | unknown | NS_ABORT_OOM | mozilla::dom::HTMLImageElement::GetCurrentSrc`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsParser::ResumeParse`
- `OOM | large | mozalloc_abort | moz_xmalloc | IPC::Channel::ChannelImpl::OnIOCompleted`
- `OOM | large | NS_ABORT_OOM | mozilla::AddonManagerStartup::EncodeBlob`
- `OOM | large | xul.dll | strset`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_text`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF16toUTF8::NS_ConvertUTF16toUTF8 | NS_MakeAbsoluteURI`
- `OOM | large | mozalloc_abort | alloc::raw_vec::impl$1::reserve::do_reserve_and_handle<T>`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::net::nsStandardURL::SetSpecWithEncoding`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::InternalBarrierMethods<T>::postBarrier`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | thread_start<T>`
- `java.lang.OutOfMemoryError: at com.sun.jna.Memory.share(Memory.java)`
- `java.lang.OutOfMemoryError: at java.lang.StringFactory.newStringFromChars(Native Method)`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Append | nsParser::ResumeParse`
- `java.lang.OutOfMemoryError: at java.util.Arrays.copyOfRange(Arrays.java)`
- `EMPTY: no crashing thread identified; StreamSizeMismatch`
- `OOM | large | mozalloc_abort | webrender::glyph_rasterizer::{{impl}}::flush_glyph_requests::{{closure}}`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::ZoneList<T>::ResizeAdd`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::InsertSlotsAt<T> | nsTArray_Impl<T>::SetLength<T> | GetPrinterInfo4`
- `OOM | large | mozalloc_abort | webrender::renderer::Renderer::update_texture_cache`
- `OOM | large | mozalloc_abort | xul.dll | patched_BaseThreadInitThunk`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Request::Constructor`
- `OOM | large | libxul.so@0x52851aa | libxul.so@0xa3e7fff | libxul.so@0xa3e7fff | libxul.so@0x5646a84 | libxul.so@0xa3e7fff | libxul.so@0x56fcdf5 | libxul.so@0x56fcddd | libxul.so@0xa3e7fff | libxul.so@0x56fcf17 | _fini`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_base<T>::InsertSlotsAt<T> | GetPrinterInfo4`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::CycleCollectedJSContext::enqueuePromiseJob`
- `OOM | large | EMPTY: no crashing thread identified; StreamSizeMismatch`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | std::vector<T>::reserve`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.GeckoAppShell.getCurrentBatteryInformation(GeckoAppShell.java)`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::AudioSink::AudioSink`
- `OOM | large | mozalloc_abort | webrender::clip::ClipStore::new`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::cyclecollector::DropJSObjectsImpl`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::irregexp::CompilePattern`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | NS_internal_main`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | SnowWhiteKiller::~SnowWhiteKiller`
- `OOM | unknown | NS_ABORT_OOM`
- `OOM | large | mozalloc_abort | webrender_bindings::bindings::wr_dp_push_hit_test`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsGenericHTMLElement::SetInnerText`
- `OOM | large | xul.dll | wmain`
- `OOM | large | mozalloc_abort | mozilla::layers::ClipManager::DefineClipChain`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsIGlobalObject::ForEachEventTargetObject`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::OrderedHashMap<T>::put<T>`
- `java.lang.OutOfMemoryError: at java.util.Arrays.copyOf(Arrays.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::StoreBuffer::MonoTypeBuffer<T>::sinkStore`
- `java.lang.OutOfMemoryError: at java.io.ByteArrayOutputStream.expand(ByteArrayOutputStream.java)`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_border`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Element::SetAttrAndNotify`
- `java.lang.OutOfMemoryError: at java.util.ArrayList.iterator(ArrayList.java)`
- `java.lang.OutOfMemoryError: at java.nio.DirectByteBuffer.<init>(DirectByteBuffer.java)`
- `OOM | large | NS_ABORT_OOM | nsStringInputStream::SerializeInternal<T>`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::FileReader::DoReadData`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::InsertSlotsAt<T> | nsTArray_Impl<T>::SetLength<T> | GetPrinterInfo4`
- `OOM | large | mozalloc_abort | webrender::renderer::Renderer::render_impl`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::dom::PerformanceEntry_Binding::get_name`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::(anonymous namespace)::BytecodeSequenceNode::FollowedBy`
- `java.lang.OutOfMemoryError: at android.view.VelocityTracker.obtain(VelocityTracker.java)`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_reference_frame`
- `EMPTY: no crashing thread identified`
- `OOM | large | mozalloc_abort | xul.dll | webrender_bindings::bindings::wr_state_new`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::MapObject::set_impl`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::SetLength<T> | GetPrinterInfo4`
- `java.lang.OutOfMemoryError: at org.mozilla.geckoview.PanZoomController$MotionEventData.<init>(PanZoomController.java)`
- `OOM | large | org.mozilla.gecko.mozglue.GeckoLoader$AbortException: at org.mozilla.gecko.mozglue.GeckoLoader.abort(GeckoLoader.java)`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_filters`
- `OOM | large | firefox-esr@0x19c0a`
- `OOM | large | NS_ABORT_OOM | mozilla::gfx::SourceSurfaceSharedDataWrapper::Map`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AssignInternal<T> | mozilla::CryptoSample::operator=`
- `OOM | large | mozalloc_abort | style::values::specified::svg_path::SVGPathData::decode_from_f32_array`
- `OOM | large | libxul.so@0x5aca8a | libxul.so@0xa305fff | libxul.so@0xa305fff | libxul.so@0x109034b | libmesa_dri_drivers.so@0x279a18`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::detail::SetValueInProxy`
- `OOM | large | NS_ABORT_OOM | mozilla::InternetCiter::Rewrap`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::ScrollFrameHelper::AppendScrollUpdate`
- `OOM | large | mozalloc_abort | xul.dll | alloc::raw_vec::{{impl}}::reserve::do_reserve_and_handle<T>`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::NativeSetProperty<T>`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::basic_string<T>::_Reallocate_for<T>`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::CheckNotCharacterAfterAnd`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::MakeEntryHandle | mozilla::dom::Timeout::SetTimeoutContainer`
- `OOM | large | NS_ABORT_OOM | xpc::ErrorReport::Init`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::jit::PostWriteBarrier`
- `OOM | large | mozalloc_abort | xul.dll | BaseThreadInitThunk`
- `EMPTY: no crashing thread identified; OK`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | wmain`
- `java.lang.OutOfMemoryError: at java.util.LinkedHashMap.createEntry(LinkedHashMap.java)`
- `java.lang.OutOfMemoryError: at dalvik.system.VMRuntime.newUnpaddedArray(Native Method)`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::DeprecationWarning`
- `OOM | large | NS_ABORT_OOM | nsGenericHTMLElement::GetURIAttr`
- `java.lang.OutOfMemoryError: at java.io.ByteArrayOutputStream.toByteArray(ByteArrayOutputStream.java)`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementInternal<T> | mozilla::ScrollFrameHelper::ScrollToImpl`
- `OOM | large | NS_ABORT_OOM | nsDataHandler::CreateNewURI`
- `java.lang.OutOfMemoryError: at java.util.LinkedList.linkLast(LinkedList.java)`
- `OOM | large | mozalloc_abort | xul.dll | webrender_bindings::bindings::wr_dp_push_text`
- `OOM | large | mozalloc_abort | moz_xrealloc | mozilla::EventListenerManager::AddEventListenerInternal`
- `OOM | unknown | NS_ABORT_OOM | mozilla::layers::SharedSurfacesParent::Add`
- `OOM | large | mozalloc_abort | alloc::raw_vec::{{impl}}::reserve::do_reserve_and_handle<T>`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::LoadCurrentCharacterImpl`
- `java.lang.OutOfMemoryError: at java.util.HashMap.makeTable(HashMap.java)`
- `OOM | large | mozalloc_abort | xul.dll | FlsGetValue`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | pref_Lookup`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::dom::Element::GetAttr`
- `OOM | large | xul.dll | _PR_NativeRunThread | pr_root`
- `OOM | large | xul.dll`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::PushBacktrack`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::SMRegExpMacroAssembler::Bind`
- `OOM | large | NS_ABORT_OOM | GatherLine`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::CellHeaderPostWriteBarrier`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | nsStringInputStream::SerializeInternal<T>`
- `OOM | large | mozalloc_abort | moz_xmalloc | SkArenaAlloc::ensureSpace`
- `java.lang.OutOfMemoryError: at java.util.ArrayList.toArray(ArrayList.java)`
- `java.lang.OutOfMemoryError: at kotlin.io.ExceptionsKt.readText(Exceptions.kt:2)`
- `OOM | large | xul.dll | <unknown in ucrtbase.dll> | xul.dll`
- `EMPTY: no crashing thread identified; EmptyMinidump`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | std::__ndk1::__tree<T>::__emplace_unique_key_args<T>`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Element::GetAttr`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::EnsureTrackPropertyTypes`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpDisjunction::ToNode`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::SparseBitmap::createBlock`
- `OOM | large | mozalloc_abort | xul.dll | rkv::backend::impl_safe::environment::impl$0::open`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsHtml5TreeOperation::Perform`
- `java.lang.OutOfMemoryError: at android.net.LinkProperties$1.createFromParcel(LinkProperties.java)`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Document::GetDocumentURI`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF8toUTF16::NS_ConvertUTF8toUTF16 | nsDocShell::OnOverLink`
- `OOM | unknown | NS_ABORT_OOM | nsIFrame::DisplayBackgroundUnconditional`
- `OOM | large | mozalloc_abort | xul.dll | core::intrinsics::const_eval_select<T>`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::pop_stacking_context`
- `java.lang.OutOfMemoryError: at java.lang.ref.FinalizerReference.add(FinalizerReference.java)`
- `OOM | large | NS_ABORT_OOM | IPC::ParamTraits<nsTSubstring<T> >::Read`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | CreatePropertyIterator`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | MimeInlineTextHTMLParsed_parse_line`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | std::vector<T>::_Reallocate_exactly`
- `java.lang.OutOfMemoryError: at org.mozilla.geckoview.GeckoSession$Window.onLoadRequest(GeckoSession.java)`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsExpatDriver::HandleError`
- `java.lang.OutOfMemoryError: at android.os.ThreadLocalWorkSource.setUid(ThreadLocalWorkSource.java)`
- `java.lang.OutOfMemoryError: at java.lang.String.fastSubstring(Native Method)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::GCPtr<T>::setUnchecked`
- `OOM | unknown | NS_ABORT_OOM | mozilla::gfx::SourceSurfaceSharedDataWrapper::Map`
- `OOM | large | libxul.so@0x59a3c6 | libxul.so@0x9065fff | libxul.so@0x59a3a5 | libxul.so@0x9065fff | libxul.so@0x13679a5 | libmediaclient.so@0x3f58`
- `java.lang.OutOfMemoryError: at java.lang.StringBuilder.toString(StringBuilder.java)`
- `OOM | large | mozalloc_abort | webrender_bindings::bindings::wr_api_end_builder`
- `OOM | large | mozalloc_abort | fog::private::timing_distribution::impl$3::stop_and_accumulate`
- `java.lang.OutOfMemoryError`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::gc::StoreBuffer::MonoTypeBuffer<T>::trace`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::SetFunctionNativeReserved`
- `OOM | unknown | TerminateProcessOnMemoryExhaustion`
- `OOM | large | xul.dll | NS_internal_main`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::a11y::DocAccessibleChildBase::SerializeTree`
- `OOM | large | xul.dll | _InternalCallWinProc`
- `java.lang.OutOfMemoryError: at android.os.Parcel.obtain(Parcel.java)`
- `OOM | large | mozalloc_abort | webrender_bindings::bindings::wr_dp_define_clipchain`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | nsLineBreaker::AppendText`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::(anonymous namespace)::BytecodeSequenceNode::MapArgument`
- `OOM | unknown | NS_ABORT_OOM | IPC::ParamTraits<nsTSubstring<T> >::Read`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF8toUTF16::NS_ConvertUTF8toUTF16 | mozilla::HTMLEditor::ParseCFHTML`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::scache::StartupCache::WriteToDisk`
- `java.lang.OutOfMemoryError: at com.sun.jna.Structure.getFieldList(Structure.java)`
- `java.lang.OutOfMemoryError: at com.sun.jna.Function.invoke(Function.java)`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsExpatDriver::ConsumeToken`
- `OOM | large | firefox-esr@0x7210a`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | nsMsgDatabase::ListAllKeys`
- `OOM | large | mozalloc_abort | moz_xrealloc | gfxUserFontEntry::SanitizeOpenTypeData`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpAtom::ToNode`
- `java.lang.OutOfMemoryError: at android.view.Choreographer.doFrame(Choreographer.java)`
- `java.lang.OutOfMemoryError: at libcore.io.IoBridge.closeAndSignalBlockedThreads(IoBridge.java)`
- `java.lang.OutOfMemoryError: at org.mozilla.geckoview.GeckoResult.<init>(GeckoResult.java)`
- `OOM | large | mozalloc_abort`
- `OOM | small`
- `OOM | large | EMPTY: no crashing thread identified; OK`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::traceObject`
- `OOM | large | xul.dll | CACPWrap::GetTextExt`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::HTMLImageElement::GetCurrentSrc`
- `OOM | large | mozalloc_abort | webrender::glyph_rasterizer::impl$1::flush_glyph_requests::closure$0`
- `OOM | large | mozalloc_abort | xul.dll | rkv::backend::impl_safe::environment::{{impl}}::open`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::SetLength | IPC::ParamTraits<nsTSubstring<T> >::Read`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::safebrowsing::ProtocolParserV2::ProcessChunk`
- `java.lang.OutOfMemoryError: at android.hardware.SystemSensorManager$SensorEventQueue.dispatchSensorEvent(SystemSensorManager.java)`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::traceObject`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | mozilla::InternetCiter::Rewrap`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::MovableCellHasher<T>::hash`
- `java.lang.OutOfMemoryError: at org.mozilla.geckoview.GeckoDisplay$ScreenshotBuilder.capture(GeckoDisplay.java)`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::net::CookiePersistentStorage::Read`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::gc::AllocateCellInGC`
- `OOM | unknown | NS_ABORT_OOM | mozilla::ArenaAllocator<T>::Allocate`
- `OOM | large | mozalloc_abort | libxul.so@0x68cfebb | libxul.so@0xa3e7fff | libxul.so@0x42639cb | libxul.so@0xa3e7fff | libxul.so@0x6919eed | libxul.so@0x6919ec8 | libxul.so@0xa3e7fff | libxul.so@0x6965a3f | libxul.so@0x6965a28 | _fini`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_rect`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::jit::CacheIRWriter::copyStubData`
- `OOM | large | NS_ABORT_OOM | mozilla::css::Loader::LoadInlineStyle`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::dom::CallbackObject::cycleCollection::DeleteCycleCollectable`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::basic_stringbuf<T>::overflow`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementInternal<T> | mozilla::dom::Performance::GetEntriesByName`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::onStringEdge`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpParser::ParseDisjunction`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::TimeRanges::Add`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.media.GeckoHlsRendererBase.getQueuedSamples(GeckoHlsRendererBase.java)`
- `OOM | large | mozalloc_abort | webrender::render_backend::Document::rebuild_hit_tester`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | _InternalCallWinProc`
- `OOM | unknown | RaiseFailFastException | TerminateProcessOnMemoryExhaustion`
- `OOM | unknown | NS_ABORT_OOM | XPCJSContext::Initialize`
- `OOM | unknown | NS_ABORT_OOM | NS_EscapeURLSpan`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpStack::EnsureCapacity`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::FragmentOrElement::CanSkipInCC`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::TypedArrayObject::objectMoved`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::GoTo`
- `java.lang.OutOfMemoryError: at kotlinx.coroutines.CancellableContinuationImpl.getResult(CancellableContinuationImpl.kt:9)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | JS::HeapObjectPostWriteBarrier`
- `java.lang.OutOfMemoryError: at android.os.Parcel.nativeCreateByteArray(Native Method)`
- `java.lang.OutOfMemoryError: at java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java)`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_item_to_section`
- `OOM | large | mozalloc_abort | webrender::scene_building::SceneBuilder::build_item`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::URL::GetHref`
- `OOM | large | libxul.so@0x5aca8a | libxul.so@0xa305fff | libxul.so@0xa305fff | libxul.so@0x109034b | libxul.so@0xa305fff | libxul.so@0xa305fff | libxul.so@0x12fbd2d | libxul.so@0x12fbd07 | libxul.so@0xa305fff | libxul.so@0x13033d7 | libxul.so@0x130321e...`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::frontend::InputName::isEqualTo`
- `OOM | large | NS_ABORT_OOM | nsParser::ResumeParse`
- `java.lang.OutOfMemoryError: at java.io.BufferedInputStream.<init>(BufferedInputStream.java)`
- `OOM | large | mozalloc_abort | moz_xrealloc | SkTDArray<T>::append`
- `java.lang.OutOfMemoryError: at dalvik.system.VMStack.getThreadStackTrace(Native Method)`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::dom::PromiseJobCallback::~PromiseJobCallback`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::onObjectEdge`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | Gecko_ContentList_AppendAll`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | nsMsgCompose::LoadDataFromFile`
- `OOM | large | libxul.so@0x5aca8a | libxul.so@0xa305fff | libxul.so@0xa305fff | libxul.so@0x109034b | libxul.so@0xa305fff | libxul.so@0x12fbe53 | libxul.so@0x12fbe37 | libxul.so@0xa0d452f | libxul.so@0xa305fff | libxul.so@0x12fc300 | libxul.so@0xa3316ff...`
- `java.lang.OutOfMemoryError: at kotlinx.coroutines.channels.ChannelsKt.cancelConsumed(Unknown Source)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | std::_Hash<T>::_Init | v8::internal::(anonymous namespace)::BytecodeSequenceNode::BytecodeSequenceNode`
- `OOM | unknown | NS_ABORT_OOM | NS_MakeAbsoluteURI`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_base<T>::InsertSlotsAt<T> | GetPrinterInfo4`
- `OOM | large | mozalloc_abort | nsNodeInfoManager::Allocate`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::GCRuntime::mergeRealms`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::gc::AllocateCellInGC`
- `OOM | large | mozalloc_abort | rayon::iter::plumbing::bridge_producer_consumer::helper<T>`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementsInternal<T> | nsHtml5TreeOpExecutor::RunFlushLoop`
- `java.lang.OutOfMemoryError: at android.os.SystemProperties.native_get(Native Method)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::Zone::New<T>(v8::internal::(anonymous namespace)::RegExpParserState*&, v8::internal::(anonymous namespace)::SubexpressionType&, v8::internal::RegExpLookaround::Type&, int&, v8::interna...`
- `OOM | large | xul.dll | <unknown in ucrtbase.dll> | xul.dll | moz_arena_malloc`
- `OOM | large | mozalloc_abort | xul.dll | nss3.dll | thread_start<T>`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::camera::DeliverFrameRunnable::DeliverFrameRunnable`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | TextToNode`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::SetLength<T> | GetPrinterInfo4`
- `OOM | large | xul.dll | IDataObject_GetData_Stub`
- `OOM | unknown | NS_ABORT_OOM | XPCJSContext::NewXPCJSContext`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsObserverList::AppendStrongObservers`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::traverse`
- `java.lang.OutOfMemoryError: at kotlinx.coroutines.CancellableContinuationImpl.resumedState(CancellableContinuationImpl.kt:4)`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::image::nsAVIFDecoder::Decode`
- `OOM | large | NS_ABORT_OOM | TextToNode`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Assign | nsTSubstring<T>::Append | Gecko_AppendCString`
- `OOM | large | NS_ABORT_OOM | nsTArrayInfallibleAllocator::SizeTooBig | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::TimeRanges::Add`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::detail::HashTable<T>::add<T> | mozilla::JSHolderMap::Put`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::GetterSetter::create`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::dom::Element::GetAttr`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::GetMemoryMappings`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::storage::Row::initialize`
- `OOM | large | mozalloc_abort | moz_xmalloc | auto_array<T>::reserve`
- `java.lang.OutOfMemoryError: at java.net.InetAddress.getByAddress(InetAddress.java)`
- `java.lang.OutOfMemoryError: at java.lang.AbstractStringBuilder.<init>(AbstractStringBuilder.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::collectToObjectFixedPoint`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::PopCurrentPosition`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::frontend::ScopeContext::searchInEnclosingScope`
- `java.lang.OutOfMemoryError: at java.lang.Long.valueOf(Long.java)`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::define_clip_rect`
- `OOM | large | mozalloc_abort | moz_xrealloc | mozilla::dom::FragmentOrElement::CanSkip`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::JSHolderMap::Put`
- `java.lang.OutOfMemoryError: at android.database.CursorWindow.nativeGetString(Native Method)`
- `java.lang.OutOfMemoryError: at android.text.StaticLayout.out(StaticLayout.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::(anonymous namespace)::BytecodeSequenceNode::BytecodeSequenceNode`
- `OOM | large | mozalloc_abort | core::intrinsics::const_eval_select<T>`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBuilder::FlushTerms`
- `java.lang.OutOfMemoryError: at android.graphics.Bitmap.nativeCreate(Native Method)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | mozilla::CycleCollectedJSContext::enqueuePromiseJob`
- `OOM | large | mozalloc_abort | webrender::renderer::upload::upload_to_texture_cache`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementsInternal<T> | nsHtml5TreeOpStage::MoveOpsFrom`
- `java.lang.OutOfMemoryError: at java.lang.Object.internalClone(Native Method)`
- `OOM | large | NS_ABORT_OOM | nsTextFrame::GetRenderedText`
- `OOM | large | mozalloc_abort | webrender_api::display_list::DisplayListBuilder::push_image`
- `OOM | large | mozalloc_abort | xul.dll | mozilla::wr::DisplayListBuilder::DisplayListBuilder`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | _PR_NativeRunThread | pr_root`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::net::CacheFileMetadata::ParseMetadata`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::SetLength | mozilla::dom::FileReader::DoReadData`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::basic_string<T>::_Construct_lv_contents`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | CCGraphBuilder::NoteWeakMapping`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | JS::HeapValuePostWriteBarrier`
- `OOM | large | mozalloc_abort | xul.dll | wmain`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::CheckNotCharacter`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::AppendElementsInternal<T> | nsHtml5TreeOpExecutor::RunFlushLoop`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::HeapPtr<T>::HeapPtr`
- `OOM | large | mozalloc_abort | style::properties::cascade::Cascade::compute_visited_style_if_needed<T>`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsTextFormatter::StringStuff`
- `OOM | large | mozalloc_abort | enum$<T>::new_mask`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::storage::Statement::GetString`
- `OOM | unknown | GraphWalker<T>::DoWalk`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Element::UnsetAttr`
- `OOM | unknown | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::DataChannelConnection::HandleDataMessage`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::Nursery::collectToObjectFixedPoint`
- `OOM | large | mozalloc_abort | xul.dll | webrender::renderer::Renderer::render_impl`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::SnappyUncompress`
- `java.lang.OutOfMemoryError: at java.util.HashSet.<init>(HashSet.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::Bind`
- `java.lang.OutOfMemoryError: at java.lang.reflect.Method.invoke(Native Method)`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::net::nsStandardURL::SetSpecWithEncoding`
- `java.lang.OutOfMemoryError: at android.net.Network$1.createFromParcel(Network.java)`
- `OOM | large | mozalloc_abort | webrender::scene_builder_thread::SceneBuilderThread::process_transaction`
- `java.lang.OutOfMemoryError: at android.view.DisplayEventReceiver.dispatchVsync(DisplayEventReceiver.java)`
- `OOM | large | js::AutoEnterOOMUnsafeRegion::crash | js::AutoEnterOOMUnsafeRegion::crash | js::TenuringTracer::onObjectEdge`
- `OOM | large | mozalloc_abort | webrender_bindings::bindings::wr_state_new`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsIFrame::DestroyFrom`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | nsPlainTextSerializer::OutputManager::Append`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::vector<T>::_Buy`
- `java.lang.OutOfMemoryError: at org.mozilla.geckoview.RuntimeTelemetry$Proxy.dispatchHistogram(RuntimeTelemetry.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::ValueToIterator`
- `OOM | large | mozalloc_abort | moz_xmalloc | ProfilingStack::ensureCapacitySlow`
- `java.lang.OutOfMemoryError: at java.lang.reflect.Constructor.newInstance(Native Method)`
- `java.lang.OutOfMemoryError: at java.lang.String.<init>(String.java)`
- `EMPTY: no crashing thread identified; HeaderMismatch`
- `OOM | large | firefox-esr@0x18388`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll`
- `OOM | unknown | XPCJSContext::NewXPCJSContext`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::vector<T>::_Emplace_reallocate<T> | mozilla::gfx::PathBuilderCairo::BezierTo`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::detail::OrderedHashTable<T>::rehash`
- `OOM | large | NS_ABORT_OOM | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::TimeRanges::Add`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::MapObject::set`
- `OOM | large | mozalloc_abort | webrender::frame_builder::FrameBuilder::build`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::gc::StoreBuffer::putSlot`
- `OOM | large | mozalloc_abort | moz_xmalloc | NS_Atomize`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::Element::SetAttr`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.util.GeckoBundle.<init>(GeckoBundle.java)`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsContentList::PopulateSelf`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::MakeEntryHandle | PLDHashTable::WithEntryHandle<T> | mozilla::dom::Timeout::SetTimeoutContainer`
- `OOM | large | mozalloc_abort | xul.dll | errno`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::Expand`
- `OOM | large | mozalloc_abort | rkv::readwrite::Writer<T>::commit<T>`
- `OOM | unknown | NS_ABORT_OOM | mozilla::jni::StringParam::GetString`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::SetLength | mozilla::dom::SnappyUncompress`
- `java.lang.OutOfMemoryError: at java.util.ArrayList.add(ArrayList.java)`
- `OOM | large | NS_ABORT_OOM | NS_ConvertUTF16toUTF8::NS_ConvertUTF16toUTF8 | nsContentUtils::GetAttrTriggeringPrincipal`
- `OOM | large | mozalloc_abort | moz_xmalloc | xul.dll | moz_xmalloc | xul.dll`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::SetLength | IPC::ParamTraits<nsTSubstring<T> >::Read`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::scache::StartupCache::GetBuffer`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsFloatManager::AddFloat`
- `OOM | unknown | NS_ABORT_OOM | mozilla::jni::ReportException`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | nsTArray_Impl<T>::InsertElementAtInternal<T> | CorpusStore::updateTrait`
- `java.lang.OutOfMemoryError: at dalvik.system.VMRuntime.newNonMovableArray(Native Method)`
- `OOM | large | xul.dll | <unknown in ucrtbase.dll> | xul.dll | SymCryptRngAesGenerate`
- `OOM | large | NS_ABORT_OOM | PrintfAppend<T>::append`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::LSValue::Converter::Converter`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::MakeEntryHandle | PLDHashTable::WithEntryHandle<T> | nsCSSFrameConstructor::NotifyDestroyingFrame`
- `OOM | large | mozalloc_abort | moz_xrealloc | xul.dll | NS_internal_main`
- `EMPTY: no crashing thread identified; MissingThreadList`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | JSONCreator`
- `OOM | large | mozalloc_abort | webrender::scene_building::SceneBuilder::build_all`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::RegExpBytecodeGenerator::WriteCurrentPositionToRegister`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::ipc::PBackgroundParent::AllManagedActors`
- `java.lang.OutOfMemoryError: at java.lang.StringFactory.newStringFromChars(StringFactory.java)`
- `OOM | large | mozalloc_abort | moz_xrealloc | gfxOTSExpandingMemoryStream<T>::WriteRaw`
- `OOM | large | NS_ABORT_OOM | NS_GetSanitizedURIStringFromURI`
- `java.lang.OutOfMemoryError: at org.mozilla.gecko.media.GeckoHlsPlayer.pause(GeckoHlsPlayer.java)`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | v8::internal::Isolate::NewByteArray`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | IPC::ParamTraits<nsTArray<T> >::Read`
- `OOM | large | mozalloc_abort | xul.dll | je_realloc | xul.dll`
- `OOM | unknown | NS_ABORT_OOM | nsPresArena<T>::Allocate`
- `OOM | unknown | <unknown in atidxx64.dll> | CContext::TID3D11DeviceContext_DrawIndexedInstanced_<T>`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::image::nsAVIFDecoder::DoDecode`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::wasm::Init`
- `OOM | large | NS_ABORT_OOM | mozilla::dom::PerformanceTimingData::Create`
- `java.lang.OutOfMemoryError: at java.lang.Integer.valueOf(Integer.java)`
- `OOM | large | mozalloc_abort | moz_xmalloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::indexedDB::PBackgroundIDBDatabaseChild::AllManagedActors`
- `java.lang.OutOfMemoryError: at android.net.Network$2.createFromParcel(Network.java)`
- `OOM | large | mozalloc_abort | webrender_bindings::bindings::wr_dp_push_text`
- `OOM | large | mozalloc_abort | moz_xmalloc | std::allocator<T>::allocate`
- `java.lang.OutOfMemoryError: at java.lang.Class.getDeclaredFields(Native Method)`
- `OOM | large | mozalloc_abort | IPC::ParamTraits<mozilla::Maybe<T> >::Read`
- `OOM | large | mozalloc_abort | moz_xrealloc | mozilla::dom::FragmentOrElement::cycleCollection::CanSkipReal`
- `OOM | large | mozalloc_abort | webrender::hit_test::add_clips`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Append | MimeRebuffer::IncreaseBuffer`
- `OOM | large | NS_ABORT_OOM | nsTSubstring<T>::Assign | mozilla::net::nsStandardURL::Resolve`
- `OOM | large | mozalloc_abort | moz_xrealloc | nsTArray_base<T>::EnsureCapacity<T> | mozilla::dom::Document::ScheduleFrameRequestCallback`
- `OOM | large | NS_ABORT_OOM | PLDHashTable::Add | gfxFontGroup::BuildFontList`
- `OOM | unknown | js::AutoEnterOOMUnsafeRegion::crash | js::jit::ExecutableAllocator::poisonCode`
- `OOM | large | mozalloc_abort | webrender::scene_building::SceneBuilder::add_primitive_to_draw_list`
- `OOM | large | mozalloc_abort | moz_xmalloc | mozilla::detail::HashTable<T>::changeTableSize | mozilla::cyclecollector::HoldJSObjectsImpl`

</details>

Reply:
@marco-c wdyt?

Reply:
I would ignore those and let developers choose the level of severity (and whether they're top crashers or not), as we can't be sure.

Reply:
Done in 805a45267a7d8cd30a5e1c6eb74f83e438f2be8f